### ISSUE TITLE
Docs beta launch communications plan

### ISSUE THREAD

**Issue by jessicaschilling** [ipfs/docs#351]
## Docs beta launch communications plan

_Note: These are taken from the instructions in the [IPFS Q4 Comms Plan](https://docs.google.com/document/d/1nk_I9YzZcnkyoK-lVLFZ-m4Qce-_JtOlfDbtGYbfY0k/edit#)._

### 3 weeks before launch _(c. 27 Nov)_
- [x] Clone and fill in the comms doc template, highlighting the core messages (new interface and nav, improved search, but content on parity with legacy site)
- [x] Share that [master comms doc](https://docs.google.com/document/d/16gC5HmP9zzQQR8vNm8baHX6x3nxQ1lallB2kqjzqDAs/edit?usp=sharing) with core WG participants and comms owners
- [x] Start drafting artifacts, with an emphasis on CTAs of contributing content and participating in user research: blog post, Reddit post, HN post, tweets, forum posts, IRC posts, canned responses, important links

### 1 week before launch _(1 Jan)_
- [x] Make sure artifacts are complete and in the comms doc: blog post, Reddit post, HN post, tweets, forum posts, IRC posts, canned responses, important links
- [x] [Create a PR on the ipfs/blog repo](https://github.com/ipfs/blog/pull/345) and request review
- [x] Confirm completion or deferral of all items in the [barebones beta launch milestone](https://github.com/ipfs/docs/milestone/3)

### 1 day before launch _(7 Jan)_
- [x] Double-check all links in blog post and comms doc are correct
- [x] Ensure CTAs are still clear, correct and actionable
- [x] Merge blog PR for automatic CI rollout (double-check the post date first!)
- [x] Merge callouts on legacy site from PR #384 

### Launch day _(8 Jan)_
- [x] Confirm blog post is live
- [x] Send to shipped@
- [x] Share remaining artifacts: Reddit post, HN post, tweets, forum posts, IRC posts

### Two weeks following launch 
- [x] Monitor response on Reddit, Twitter, HN, forum, IRC and respond accordingly

**Comment by jessicaschilling** [comment]
TLDR: Comms plan is here: https://docs.google.com/document/d/16gC5HmP9zzQQR8vNm8baHX6x3nxQ1lallB2kqjzqDAs

**Comment by jessicaschilling** [comment]
Assigning @johnnymatthews to this issue as well, since he's offered to draw up a mind map of the new/updated content plan that can be used in the blog post (and elsewhere as possible).

**Comment by johnnymatthews** [comment]
Map should be finished in the next couple of days. There's a lot of figuring out where current posts go, where the holes are, and how to fill them in.

**Comment by terichadbourne** [comment]
Just popping in to say that this comms plan looks great. ❤️

**Comment by jessicaschilling** [comment]
@johnnymatthews -- I've moved the mind map into issue #389 since it's maybe a little early-stage for the blog post, but SUPER useful for that issue. Thank you!

**Comment by johnnymatthews** [comment]
Mind map of the _current docs_ hierarchy and content.

![Current Map](https://user-images.githubusercontent.com/9611008/70445826-30743880-1a6a-11ea-8425-fb0af4d296a5.png)

**Comment by jessicaschilling** [comment]
@johnnymatthews This is awesome! Thanks.

**Comment by jessicaschilling** [comment]
Update: Remainder of plan will be delayed pending the final outcome of IPFS hosting decisions discussed in design review tomorrow; see #271 for progress on that. However, all the materials in the comms plan are ready to ship as soon as we confirm a ship date for the docs themselves.

**Comment by jessicaschilling** [comment]
Dates changed to January pending directive given regarding #271.

**Comment by jessicaschilling** [comment]
Launch complete and all artifacts shared. I have space allocated in my calendar for the next two weeks for monitoring all the channels above, and will comment on this issue if anything relevant needs capturing -- but will close the issue in the interim.
